protobuf-format.md

```diff
@@ -0,0 +1,88 @@
+# Protobuf Event Format for CloudEvents  - Version 0.1
+
+## Abstract
+
+The Protocol Buffers Format for CloudEvents (CE) defines
+the encoding of events in the protocol buffers binary format.
+
+## Status of this document
+
+This document is a working draft.
+
+## 1. Introduction
+
+CloudEvents is a standardized and transport-neutral definition of the
+structure and metadata description of events. This specification
+defines how the [Context
+Attributes](spec.md#context-attributes) defined in the CloudEvents in
+the protocol buffers binary encoding MUST BE encoded. Transcoding to
+and from other formats (e.g. JSON) is out of the scope of this
+proposal.
+
+Protocol buffers are a language-neutral, platform-neutral extensible
+mechanism for serializing structured data. The [Google reference
+implementation of
+protobuf](https://github.com/protocolbuffers/protobuf) includes
+support for an interface descriptor language (IDL), and this document
+makes use of language level 3 IDL from Protocol Buffers
+v3.5.0. CloudEvents systems using protocol buffers are not required to
+use the IDL or any particular implementation of Protocol buffers as
+long as they produce messages which match the binary encoding defined
+by the IDL.
+
+
+### 1.1. Conformance
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in RFC2119.
```

Comment:
can you make the RFC2119 word into a link to https://tools.ietf.org/html/rfc2119  ?